recipe/meta.yaml

```
@@ -7,46 +7,52 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytorch_tabnet-{{ version }}.tar.gz
-  sha256: 18887b993a8bf86ec05a576b5cf93f09e08b778cd9f418c5b254b6566df673a5
+  url: https://github.com/dreamquark-ai/tabnet/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 0703d0298bf605dae289515023859428017fe86e4663de9068a99f06b9802c94
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv --no-build-isolation --no-deps
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - wheel
+    - python
     - poetry >=0.12
   run:
-    - python >=3.6
-    - numpy >=1.17,<2.0
+    - python
+    - numpy {{ numpy }}
```

```suggestion
    - numpy >=1.17
```
Using the cbc version is for when numpy is used in host and the C-API is used. Here we can stick to what upstream defines.